File: src/main/kotlin/unizar/labis/g03/backendapp/domain/services/CalculoOcupantesMaximos.java
package unizar.labis.g03.backendapp.domain.services;

import unizar.labis.g03.backendapp.domain.model.entities.Espacio;

import java.util.List;

public class CalculoOcupantesMaximos {

    public Integer  maximosOcupantesValido( List<Espacio> espacios){
        int suma = 0;
        for (Espacio espacio : espacios) {
            suma += espacio.getCapacidadMaxima();
        }
        return suma;
    }
}

File: src/main/kotlin/unizar/labis/g03/backendapp/domain/services/CalculoOcupantesMaximos.kt
package unizar.labis.g03.backendapp.domain.services

import org.springframework.stereotype.Service
import unizar.labis.g03.backendapp.domain.model.entities.Espacio

@Service
class CalculoOcupantesMaximos {
    fun maximosOcupantesValido(espacios: List<Espacio>): Int {
        var suma = 0
        for (espacio in espacios) {
            suma += espacio.getCapacidadMaxima()
        }
        return suma
    }
}
